common/templates/mathjax_include.html

```diff
@@ -5,32 +5,3 @@
 ##
 ## This enables ASCIIMathJAX, and is used by js_textbox
 
-
-%if mathjax_mode is not Undefined and mathjax_mode == 'wiki':
```

Comment:
I'm being a little skeptical about this. Can't we disable it in another way? There has an attempt at the hackathon to do this as a togglable feature!